src/resources/ApiKeys/ApiKeys.ts

```diff
@@ -31,4 +31,12 @@ export default class ApiKey extends Resource {
     delete(apiKeyId: string) {
         return this.api.delete(`${ApiKey.baseUrl}/${apiKeyId}`);
     }
+
+    extend(apiKeyId: string) {
+        return this.api.put(`${ApiKey.baseUrl}/${apiKeyId}/activation/extend`);
+    }
+
+    duplicate(apiKeyId: string) {
```

Comment:
You're not using `DuplicateApiKeyOptions` here?

Reply:
This is what happen when you want a pr to be ready for a scrum. Sorry I'll go through all the comments